.gitignore

```diff
@@ -13,3 +13,46 @@ docs/.vitepress/cache
 !api/.env.sample
 **.log
 .vscode
+
+__pycache__/
```

Comment:
On folder structure I was thinking keep core and cli separately.

```
- app/main.py # typer app entry point
- app/commands
- app/utils
- core/
- tests/
- pyproject.toml
- main.py # run as python script..
```


cli/main.py => Typer App config and wiring up the commands
main.py => main entry point

Reply:
app or src, whatever convention suites i think